clients/python/build/prepare-release.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# Script for updating version numbers in preparation for a release.
+# Usage: ./prepare_release.sh <release_version>
+# Example: ./prepare_release.sh 1.2.3
+
+set -e
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BOLD='\033[1m'
+NC='\033[0m'
+
+usage() {
+  echo -e "${BOLD}Usage:${NC} $0 <release_version>"
+  echo -e "${BOLD}Example:${NC} $0 1.2.3"
+  exit 1
+}
+
+update_version_in_file() {
```

Comment:
can you add a few comments explaining what does each function in this release script do? shell scripts are not the easiest to read and understand.

Reply:
Added comments and gave a much simpler guide for release at the top of the maintainer doc